feat(changelog): kubernetes-added-kubernetes-version-1.27-now 2023-05-02 (#1610)
scaleway/docs-content: changelog/may2023/2023-05-02-kubernetes-added-kubernetes-version-1.27-now.mdx

---

> title: Kubernetes version 1.27 now available!

Reviewer: ```suggestion
title: Kubernetes version 1.27 is now available!
```

status: added
author:
    fullname: 'Join the #k8s channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-05-02
category: compute
product: kubernetes
---

> Scaleway Kubernetes products now support the latest minor version 1.27 « Chill Vibes ». Find more details in [https://scaleway.com/en/docs/containers/kubernetes/reference-content/version-support-policy](our version support policy).

Reviewer: ```suggestion
Scaleway Kubernetes products now support the latest minor version 1.27 "Chill Vibes". Read [our version support policy](/containers/kubernetes/reference-content/version-support-policy) for more information.
```
